appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockHVLayoutBase.java

```diff
@@ -317,6 +324,10 @@ private void layoutVertical(LayoutInfo containerLayoutInfo) {
     int usedHeight = 0;
     int countFillParent = 0;
     for (MockComponent child : containerLayoutInfo.visibleChildren) {
+      // skip menu when calculating height constraints
+      if (child.isMenu() || child instanceof MockFloatingActionButton) {
```

Comment:
For FAB you still need to set its position. I think what you'll want to do is in the `if` statement below to check for FAB is to compute its position and then reset the width/height of childLayoutInfo to 0.